pip/qsharp/_qsharp.py

```diff
@@ -37,16 +37,54 @@ def _repr_mimebundle_(
         return {"application/x.qsharp-config": self._config}
 
 
-def init(target_profile: TargetProfile = TargetProfile.Unrestricted) -> Config:
+def init(
+    *,
+    target_profile: TargetProfile = TargetProfile.Unrestricted,
+    project_root: Optional[str] = None,
+) -> Config:
     """
     Initializes the Q# interpreter.
 
     :param target_profile: Setting the target profile allows the Q#
         interpreter to generate programs that are compatible
         with a specific target. See :py:class: `qsharp.TargetProfile`.
+
+    :param project_root: The root directory of the Q# project. It must
+        contain a qsharp.json project manifest.
```

Comment:
nitpick: the comment makes it seem like this is required, even the parameter is optional. Maybe something like:


```suggestion
    :param project_root: An optional path to a root directory with a Q# project to include.
        It mustcontain a qsharp.json project manifest.
```